tracee-ebpf/tracee/write_capture.go

```diff
@@ -33,6 +33,11 @@ func (t *Tracee) processFileWrites() {
 		Size  uint64
 	}
 
+	type bufferModuleMeta struct {
+		Pid  uint32
+		Size uint64
+	}
```

Comment:
Can't we just use kernelModuleMeta for this as well?
If you will set both DevID and Inode to zero (from bpf code), all of the below logic won't be needed